### PR TITLE
Full review of repository code by Mariia Husak

### PR DESCRIPTION
**Pull request by MariaHusak**

### CODE REVIEW

File: BinanceXchange/settings.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+CORE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-yz@arq$5207orb+c0+5jto_#9-l==yw^#vhjw$0vz2_!!=@fer'
```
Comment (MariaHusak) [inline BinanceXchange/settings.py:26]: SECRET_KEY is better to hide

File: BinanceXchange/settings.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+
+LOGIN_REDIRECT_URL = '/'
+LOGOUT_REDIRECT_URL = '/'
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'xchangeua@gmail.com'
+EMAIL_HOST_PASSWORD = 'bcqv mmid toxp dxiq'
```
Comment (MariaHusak) [inline BinanceXchange/settings.py:178]: EMAIL_HOST_PASSWORD is better to hide

File: BinanceXchange/settings.py [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+from pathlib import Path
+from django.contrib import staticfiles
+import os
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+CORE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
```
Comment (MariaHusak) [inline BinanceXchange/settings.py:21]: bhjbhbhgbghvbgh

File: BinanceXchange/test.py
Comment (MariaHusak) [inline BinanceXchange/test.py:1]: You should add more tests.

File: templates/create_order.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+        body {
```
Comment (MariaHusak) [inline templates/create_order.html:10]: There are too many styles in html files, maybe they should be moved to css files

**Comment by MariaHusak** [comment]
The site on the remote host works fine. No critical bugs were found. The code is written simply and clearly. But you should add more tests. And take care of a better structure of html files. Overall, the code presented here is well structured and follows Django development best practices.

**Comment by dem-volodymyr** [comment]
Thanks for the review, I will take into account the notes indicating problems)

**Pull request merged**